test/tests/components/Dropdown.js

```diff
@@ -0,0 +1,72 @@
+import expect from 'expect';
+import Dropdown, { Props } from '../../../src/dropdown';
+import { newComponent } from '../helpers';
+
+describe('Dropdown', function () {
+
+  describe('locals', () => {
+    const exampleProps = {
+      id: '12345',
+      className: 'fancy-class-name',
+      style: { margin: 10, position: 'relative' },
+      value: { value: 'test', label: 'Test' },
```

Comment:
I think you should add other custom properties to this object. We're testing that `Dropdown` doesn't touch a `value` object and that should be true for **any** object even if malformed.